### ISSUE TITLE
in v5, serial COM ports are not sorted in order in New connection dialog.

### ISSUE THREAD

**Issue by toipaz** [TeraTermProject/teraterm#270]
In v5, serial COM ports are not sorted in order in New connection of serial as follows. 
![serial port list in v5 of teraterm](https://github.com/user-attachments/assets/512bc601-51eb-44bf-a0ff-be8397c8bc80)

The picture shows them sorted in order in v4.
![serial port list in v4 of teraterm](https://github.com/user-attachments/assets/f1d7333c-0df3-4895-80fc-a13c9ac8c2c0)

After installing v5, the order of COM ports in New connection was different from v4, so I ended up deleting it and installing v4.
I hope it will be fixed in v5.
I am currently using v4 just fine.
Thank you.

**Comment by zmatsuo** [comment]
Thanks for the report. @toipaz 

com port list is sorted.
But there seems to be a bug.

Please let me know the result of lscom.exe

[lscom.zip](https://github.com/user-attachments/files/16574219/lscom.zip)

This is result on my PC.
[lscom.txt](https://github.com/user-attachments/files/16574250/lscom.txt)

**Comment by toipaz** [comment]
When I ran the provided lscom, a warning message was displayed.
![Caution](https://github.com/user-attachments/assets/df9a7064-4223-4687-9606-68203762acab)

I was worried, but I ran it believing that there would be no problem.
(When providing it, it seems that it needs to be mentioned or supplemented in advance.)
I am attaching the result of running lscom.
[lscom.txt](https://github.com/user-attachments/files/16579641/lscom.txt)
I hope for good results.
Thanks
